agent/engine/docker_task_engine.go

```diff
@@ -858,8 +858,8 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	}
 
 	// apply secrets to container.Environment
-	if container.ShouldCreateWithSSMSecret() {
```

Comment:
is `ShouldCreateWithSSMSecret` used somewhere else? if not we can remove it

Reply:
There are multiple places using it. For example, line 537 in task.go